src/extension/default_extension_ek.cairo

```diff
@@ -145,7 +160,7 @@ mod DefaultExtensionEK {
                 tokenization::{tokenization_component, tokenization_component::TokenizationTrait}
             }
         },
-        vendor::erc20::{IERC20MetadataDispatcher, IERC20MetadataDispatcherTrait},
+        vendor::erc20::{ERC20ABIDispatcher as IERC20Dispatcher, ERC20ABIDispatcherTrait}, units::INFLATION_FEE,
```

Comment:
I reverted back to ERC20ABIDispatcher because that is already now used for token approvals.